app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
+    ```
+    
+8. Verify you get an OK check. You should have an output similar to this:
+ 
+    ```
+    kong-enterprise-edition-1.3.0.1.el7.noarch.rpm: sha1 md5 OK
+    ```
 
-**Kong Enterprise** on Amazon Linux 2 supports:
+### Option 2: Download the Kong Repo File and Add to Yum Repo
+   
+1. Click this URL to download the Kong Enterprise RPM repo file: https://bintray.com/kong/kong-enterprise-edition-aws/rpm.
 
-- [PostgreSQL](/#postgresql)
-- Cassandra
+2. Edit the repo file using your preferred editor and alter the baseurl line as follows::
+    
+    ```
+    baseurl=https://USERNAME:API_KEY@kong.bintray.com/kong-enterprise-edition-aws
+    ```
+  
+    - Replace `USERNAME` with your Bintray account user name.
+    - Replace `API_KEY` with your Bintray API key. You can find your key on your Bintray profile page at https://bintray.com/profile/edit and selecting the API Key menu item.
 
+    The result should look something like this:
 
-### PostgreSQL
+    ```
+    baseurl=https://john-company:12234e314356291a2b11058591bba195830@kong.bintray.com/kong-enterprise-edition-aws
+    ```
 
-This configuration supports PostgreSQL 9.6, 10, and 11 via [Amazon Linux Extras](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/amazon-linux-ami-basics.html#extras-library).
+3. Securely copy the changed repo file to your home directory on the Amazon Linux 2 system. You may use a command such as:
 
+    ```bash
+    $ scp bintray--kong-kong-enterprise-edition-aws.repo <amazon user>@<server>:~
+    ```
+    
+### Download your Kong Enterprise License
+   
+1. Download your license file from your account files in Bintray: 
+  
+    ```
+    https://bintray.com/kong/<YOUR_REPO_NAME>/license#files`
+    ```
 
-**Steps for this section:**
+2. Securely copy the license file to your home directory on the Amazon Linux system. You may use a command like:
 
-- i. [Install and Start Postgres](/#install-and-start-postgresql)
-- ii. [Create a Kong user and database](/#create-a-kong-user-and-database)
-- iii. [Connect Kong to the Postgres database](/#connect-kong-to-the-postgresql-database)
+    ```bash
+    $ scp license.json <amazon username>@<server>:~
+    ```
 
+### Result
 
-**i. Install and Start PostgreSQL**
+You should now have two files in your home directory on the target Amazon system:
 
-The examples below will use the most up-to-date version of Postgres, Postgres 11. To use 9.6 or 10, append "96" or "10" to every `postgres` command. 
+- Either the Kong RPM or Kong Yum repo file.
+- The license file `license.json`
 
-1. Install PostgreSQL
-  ```
-  $ sudo amazon-linux-extras install postgresql9.6
-  ```
-  or
-  ```
-  $ sudo amazon-linux-extras install posgresql10
-  ```
-  or
-  ```
-  $ sudo amazon-linux-extras install postgresql11
-  ```
+## Step 2. Install Kong Enterprise
 
-2. Install the PostgreSQL Server
-  ```
-  $ sudo yum install postgresql postgresql-server
-  ```
+### Option 1: If installing via a downloaded RPM package
```

Comment:
Change "via" to "using"

Reply:
I hope we add a style guide rule banning "via" 😆